utils/graphql-schemas/generate-schema-docs.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const { fetchNRGraphqlResults } = require('../nr-graphql-helpers');
+const { fullSchemaQuery } = require('./constants');
+const { renderSchema } = require('graphql-markdown');
+
+/**
+ * Print function to pass to renderSchema, writes each line separately to a file
+ * @param {String} data - The data to write to the file
+ */
+const printData = (data) => {
+  fs.appendFileSync('./graphql-schemas/schema-docs.md', `${data}\n`);
+};
+
+/**
+ * Filters the data to only include NR1Catalog endpoints
+ * @param {Object} data - The data to filter
+ * @returns {Object} The filtered data in schema form for graphql-markdown
+ */
+const filterData = (data) => {
+  const {
+    data: {
+      __schema: { types },
+    },
+  } = data;
+  const endpoints = types.filter((type) => type.name.startsWith('Nr1Catalog'));
```

Comment:
This seems to include a lot of the extra stuff under the `Nr1Catalog` namespace, maybe we could filter it down a bit more so it's just install plans and quickstarts?
Something like this might work to filter this down to just what we want
```suggestion
  const endpoints = types.filter((type) => type.name.startsWith('Nr1CatalogQuickstart') || type.name.startsWith('Nr1CatalogInstall'));
```

Reply:
I guess... ` && type.name !== 'KitchenSink'` 😂